apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/MessageError.java

```diff
@@ -113,7 +113,10 @@ public PK getPK() {
   @NoArgsConstructor
   @AllArgsConstructor
   public static class PK implements Serializable {
+    /** sequenceNumber. */
```

Comment:
Something a little more readable would be helpful for the new values here. "The sequence number for the primary key." (Particularly when the inner class is named so succinctly here)